MAIN/GPTCommunicator.py

```diff
@@ -0,0 +1,34 @@
+# This class will handle communication with the GPT-3.5 AI model and sending requests to the OpenAI API.
+import openai
+import asyncio
+import os
+from dotenv import load_dotenv
+
+# Load environment variables from .env file
+load_dotenv()

```

Comment:
Expressions outside of a function are a bad habit. This code will be called every time the file is loaded and imported.   Read more here: https://www.geeksforgeeks.org/python-main-function/amp/